### PR TITLE
Replace tito by make, unify all roles install proc

### PR DESCRIPTION
**Pull request by GAZDOWN**
Hi, this PR is replacing tito builds by make build/install as well as it is adding this unified building into all other roles.

### CODE REVIEW

File: ansible/roles/actors/defaults/main.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ actors:
         version: "{{ lookup('env', 'LEAPP_ACTORS_GIT_VERSION')|default('HEAD', true) }}"
         dest: "{{ common.paths.workdir }}/leapp-actors/"
     paths:
-        actors: "{{ common.paths.workdir }}/leapp-actors/src/actors/"
-        schemas: "{{ common.paths.workdir }}/leapp-actors/src/schemas/"
+        actors: "/usr/share/leapp/actors/"
+        schemas: "/usr/share/leapp/schemas/"
```
Comment (artmello) [inline ansible/roles/actors/defaults/main.yml]: Not necessary, but maybe place "/usr/share/leapp" into a actors.something variable

File: ansible/roles/actors/tasks/main.yml [hunk 2]
```diff
@@ -21,6 +21,17 @@
   with_items: "{{ actors.packages }}"
   when: ansible_distribution == 'CentOS'
 
+- name: "Install Leapp-Actors dependencies"
+  become: true
+  shell: "cd {{ actors.git.dest }} && make install-deps"
```
Comment (artmello) [inline ansible/roles/actors/tasks/main.yml]: This seems to work, but maybe replace the shell module for the make one. It is flagged as "preview" but seems to work on my tests. Would be something like this:
```
make:
  chdir: {{ actors.git.dest }}
  target: install-deps
``` 
Maybe even place the targets inside a for each loop

File: ansible/roles/actors/defaults/main.yml (hunk 1 above)
Comment (GAZDOWN) [reply ansible/roles/actors/defaults/main.yml]: This makes sense, thx

File: ansible/roles/actors/tasks/main.yml (hunk 2 above)
Comment (GAZDOWN) [reply ansible/roles/actors/tasks/main.yml]: I am not sure whether we really want to do this. Did you test it in Centos as well?

**Review by artmello — APPROVED** [review]
lgtm

**Pull request merged**